core/pkg/service/flag-evaluation/flag_evaluator.go

```diff
@@ -322,11 +322,9 @@ func errFormat(err error) error {
 		return connect.NewError(connect.CodeNotFound, fmt.Errorf("%s, %s", ErrorPrefix, err.Error()))
 	case model.TypeMismatchErrorCode:
 		return connect.NewError(connect.CodeInvalidArgument, fmt.Errorf("%s, %s", ErrorPrefix, err.Error()))
-	case model.FlagDisabledErrorCode:
-		return connect.NewError(connect.CodeUnavailable, fmt.Errorf("%s, %s", ErrorPrefix, err.Error()))
 	case model.ParseErrorCode:
 		return connect.NewError(connect.CodeDataLoss, fmt.Errorf("%s, %s", ErrorPrefix, err.Error()))
-	case model.GeneralErrorCode:
+	case model.FlagDisabledErrorCode, model.GeneralErrorCode:
```

Comment:
You could argue that a disabled flag should behave like it's not there. If that's the case, the `FLAG_NOT_FOUND` error code would be more appropriate. What do you think @toddbaert?

Reply:
We could also consider adding a `FLAG_DISABLED` error code but I think that would be a breaking spec change.

Reply:
Good point, I don't have a strong opinion here. I am fine with the current approach (error with FLAG_DISABLED message section) or the proposed change.

Reply:
Personally I'm of the opinion that flags disabled in flagd should behave EXACTLY as flags that aren't there. Basically they're "unreleased" flags.

Reply:
Reason will be an error and we can do the following, 

At flagd, mark disabled flags with connect `CodeNotFound` with reason (error message) `FLAG_DISABLED`

This from Go SDK will get converted to following error message, 

`openfeature: FLAG_NOT_FOUND: not_found: FlagdError:, FLAG_DISABLED`

cc @beeme1mr  @toddbaert

Reply:
@toddbaert  thanks and done - updated with the latest commit . Latest error log look like below,

`openfeature: FLAG_NOT_FOUND: not_found: FlagdError:, FLAG_DISABLED`